lib/ansible/module_utils/basic.py

```diff
@@ -2087,13 +2098,19 @@ def run_command(self, args, check_rc=False, close_fds=True, executable=None, dat
             # the communication logic here is essentially taken from that
             # of the _communicate() function in ssh.py
 
-            stdout = ''
-            stderr = ''
+            stdout = b('')
+            stderr = b('')
```

Comment:
It shouldn't change the return type.  Remember, this is python2 code being ported to run on python3 in addition.